CHANGELOG.md

```diff
@@ -212,6 +212,9 @@ Adding a new version? You'll need three changes:
   [#4969](https://github.com/Kong/kubernetes-ingress-controller/pull/4969)
   [#4979](https://github.com/Kong/kubernetes-ingress-controller/pull/4979)
   [#4989](https://github.com/Kong/kubernetes-ingress-controller/pull/4989)
+- KIC now specifies its `UserAgent` when communicating with kube-apiserver
```

Comment:
```suggestion
- KIC now specifies its `User-Agent` when communicating with kube-apiserver
```

Reply:
I believe we can leave this as is since:

- the field that we set in controller runtime config is `UserAgent`
- and field in audit logs is called `userAgent`

Even knowing that the relevant header name is `User-Agent`.